### ISSUE TITLE
new method to get hostnames

### ISSUE THREAD

**Issue by roccomuso** [tomas/network#15]
It would be worth having a method like `network.get_hostnames(cb)`, that returns a map `Hostname <--> IP` for the current active interface? (parsing the output of `arp -a` on unix).

**Comment by tomas** [comment]
You mean like parsing `/etc/hosts`?

**Comment by roccomuso** [comment]
Not really. For current machine hostname you have `os.hostname()`, `arp -a` provides LAN devices hostnames

**Comment by tomas** [comment]
Hey @roccomuso. It's been a while, but I was wondering if you're up for submitting a PR for this.

**Comment by roccomuso** [comment]
Yep, I'm wondering if it's worth creating a new module using `ip n` as fallback for `arp -a`.
Moreover the hostname is not guaranteed in the arp table. Most of the times you'll have a simply IP <-> Mac address bind.
